src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,86 @@
+import classNames from 'classnames';
+import { FC } from 'react';
+import { Link, useSearchParams } from 'react-router-dom';
+import { getSearchWith } from '../../helpers/searchHelper';
+import arrowLeft from '../../assets/svg/arrowLeft.svg';
+import arrowRigth from '../../assets/svg/arrowRigth.svg';
+import './pagination.scss';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+};
+
+export const Pagination: FC<Props> = ({ total, perPage, currentPage }) => {
+  const [searchParams] = useSearchParams('');
+  const lastPage = Math.ceil(total / perPage);
+  const numberOfPages = [];
+
+  for (let n = 1; n <= lastPage; n += 1) {
+    numberOfPages.push(n);
+  }
+
+  return (
+    <div className="pagination">
+      <Link
+        to={{
+          search: getSearchWith(searchParams, {
+            page: (currentPage - 1).toString(),
+          }),
+        }}
+        className={classNames(
+          'pagination__item',
+          'button-square',
+          {
+            'button-square--disabled': currentPage === numberOfPages[0],
+          },
+        )}
+      >
+        <img src={arrowLeft} alt="arrowLeft" />
+      </Link>
+
+      <ul className="pagination__list">
+        {numberOfPages.map((n) => {
+          const numberPage = n.toString();
+
```

Comment:
It could be remove, cuz it already will be string

Reply:
No, it couldn`t.